app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -98,3 +98,11 @@ select.date
 
 .hidden
   display: none
+
+.text-color-for-status
```

Comment:
yep @diogob actually these classes were already there

rejected & failed                            -->     .text-error
waiting_funds                                -->     .text-waiting
approved, online e succesfull       -->      .text-success

maybe @devton didn't know about that